### DISCUSSION TITLE
Documents Referenced in Source

### DISCUSSION THREAD

**Discussion by geoffroutledge** [EricBerendsen/dvbinspector#79] [General]
Eric,

I'm trying to analyse a stream which I believe to contain OpenTV data sections. I notice that in the source code (**Descriptor.java**) the comments very usefully mention the documentation used as a reference for some of the descriptors. In particular, you reference a couple of documents - "http://download.tdconline.dk/pub/kabeltv/pdf/CPE/Rules_of_Operation.pdf" and "http://www.nordig.org/pdf/NorDig_RoOspec_0_9.pdf". These look like they could be useful documents but, unfortunately, the links are dead. I don't suppose you would happen to know of an alternate source?

By the way, excellent piece of code and I really appreciate the comments in there.

Geoff.

**Comment by EricBerendsen** [comment]
Hi Geoff,

Unfortunately I don't have a copy of  those documents. Only found youseee Rules_of_Operation 1.13.pdf which describes the same descriptors. However there is not a lot of detail, so I am afraid it will not help you very much. 
[Rules_of_Operation 1.13.pdf](https://github.com/user-attachments/files/17684494/Rules_of_Operation.1.13.pdf)

**Reply by geoffroutledge** [reply]
Eric,
Thank you for taking the time to reply and for the attached document. I thought this might be the case - there doesn't seem to be a lot of public information regarding OpenTV. If I do happen to find anything public, I'll update this thread.
Best regards,
Geoff.
